engine/runtime/src/main/scala/org/enso/interpreter/instrument/execution/ErrorResolver.scala

```diff
@@ -39,7 +39,10 @@ object ErrorResolver {
     node.map(x => {
       x.getEncapsulatingSourceSection match {
         case null =>
-          Api.StackTraceElement(x.getRootNode.getName, None, None, None)
+          x.getRootNode match {
+            case null => null
```

Comment:
I would change that to `node.flatMap(... ` and return `None`/`Some` as appropriate with a guard
```
case null if x.getRootNode == null => None
case null => Some(Api.Stack...)
case section => Some(...)
``` 
and not do filtering